README.md

```diff
@@ -1,17 +1,41 @@
 # ImageProcessing
 
-Provides higher-level image processing functionality that is commonly needed
-when accepting user uploads. Supports processing with [libvips] and [ImageMagick].
+Provides higher-level image processing helpers that are commonly needed
+when handling image uploads.
+
+This gem can process images with either [libvips] and [ImageMagick] libraries.
+ImageMagick is a good default choice, especially if you are migrating from
+another attachment gem that uses ImageMagick. Livips is a very fast alternative
+that works best with JPEGs and PNGs. Support for GIFs is limited in libvips.
+
+
+## Goal
+
+The goal of this project is to have a single gem that contains all the
+helper methods needed to resize and process images. Currently, existing
+attachment gems (like Paperclip, CarrierWave, Refile, Dragonfly, 
+ActiveStorage, and others) implement their own custom image helper methods.
+But why?
+
+Let's be honest. Image processing is a dark, mysterious art. So we want to 
+combine everything bit of best practice from all of these gems into a single, 
+awesome library that is constantly updated with best-practice thinking about 
+how to resize and process images.
 
-The goal of this project is to have a single place where common image
-processing helper methods are maintained, instead of Paperclip, CarrierWave,
-Refile, Dragonfly and ActiveStorage each implementing their own versions.
 
 ## Installation
 
-```rb
-gem "image_processing", "~> 0.11"
-```
+1. Install ImageMagick and Libvips:
+
+`$ brew install imagemagick vips`
+
+Note: if you're not on macOS or don't want to use Homebrew, check the project
+pages for ImageMagick and Libvips for other ways to install these libraries.
+
+2. Add the gem to your Gemfile:
+
+`gem 'image_processing', '~> 0.11'`
```

Comment:
I would use a code block here, so that it's syntax-highlighted.

Reply:
I think we can stick with backticks for these two lines, because the github syntax highlighter for ruby and bash ignores them:

```ruby
test = "string"
gem 'image_processing', '~> 0.11'
```

```bash
cd "Desktop"
brew install imagemagick vips
```